### PR TITLE
Fix changed RuboCop methods

### LINKED ISSUE #435: Test is failed with RuboCop HEAD
A method signature of `autocorrect_source` has been changed  by https://github.com/bbatsov/rubocop/pull/4510 .
So the test for rubocop-rspec is failed with next version RuboCop.

This problem can be avoided by one of the following methods.

1. Drop rubocop 0.49.1 or older support if rubocop 0.50.0 is released
1. Pin rubocop version with 0.49.1 or older
1. Check `Method#arity` for `autocrrect_source` method.
  - This way supports both rubocop versions, but it's a workaround.

### PR DESCRIPTION
**Pull request by bquorning**
Fixes #435 :crossed_fingers:

### CODE REVIEW

File: lib/rubocop/cop/rspec/repeated_example.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@ def example_signature(example)
           key_parts = [example.metadata, example.implementation]
 
           if example.definition.method_name == :its
-            key_parts << example.definition.method_args
+            key_parts << [example.definition.first_argument]
```
Comment (bquorning) [inline lib/rubocop/cop/rspec/repeated_example.rb]: I’m not too sure the new code is equivalent to the old.

Comment (backus) [inline lib/rubocop/cop/rspec/repeated_example.rb]: if this is a block type I think you could also destructure and use the `method_args` method I think it is still there on `SendNode` types

Comment (bquorning) [inline lib/rubocop/cop/rspec/repeated_example.rb]: `example.definition.arguments` seems to work.

Comment (backus) [inline lib/rubocop/cop/rspec/repeated_example.rb]: Nice

**Comment by bquorning** [comment]
Green build, now using RuboCop v0.50.0.

**Comment by backus** [comment]
Great. If you'd like to do another release go ahead

**Pull request merged**